charts/rancher-eks-operator/1.0.600/templates/eksclusterconfig.yaml

```diff
@@ -2,8 +2,6 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   name: eksclusterconfigs.eks.cattle.io
-  annotations:
-    "helm.sh/resource-policy": keep
```

Comment:
why is this being removed?

Reply:
I ran go generate in `eks-operator` and copied this over.